modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -86,14 +87,14 @@ def generate_show_output(bgs_claim:, lighthouse_claim:) # rubocop:disable Metric
             }
           else
             bgs_details = bgs_claim[:benefit_claim_details_dto]
-            structure = build_claim_structure(data: bgs_details,
+            structure = build_claim_structure('show',
+                                              data: bgs_details,
                                               lighthouse_id: nil,
                                               upstream_id: bgs_details[:benefit_claim_id])
           end
           structure.merge!(errors: get_errors(lighthouse_claim))
           structure.merge!(supporting_documents: build_supporting_docs(bgs_claim))
           structure.merge!(tracked_items: map_bgs_tracked_items(bgs_claim))
-          structure.merge!(build_claim_phase_attributes(bgs_claim, 'show'))
```

Comment:
We were already calling `build_claim_phase_attributes` via `build_claim_structure`, so removed this duplicate call.

Reply:
@mchristiansonVA Everything looks good! One request: can we add a test to the request spec that uses the [bgs_claim_details_dto_with_phaseback_lc_status](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/spec/factories/auto_established_claims.rb#L189) factory?